lib/community/solicitor.js

```diff
@@ -46,13 +57,17 @@ var queueEmail = function (emailData) {
 }
 
 Solicitor.sendToUser = function (user, community) {
-  return queueEmail(emailData(user, community))
+  if (user.get('settings').email_solicitation_preference) {
+    return queueEmail(emailData(user, community))
+  } else {
+    return Promise.resolve()
+  }
```

Comment:
Do you have a preference for the Promise.method way?
The above is easy for me to read